src/features/hotspots/setup/HotspotSelectionScreen.tsx

```diff
@@ -28,7 +28,8 @@ const HotspotSetupSelectionScreen = () => {
           <HotspotSelectionCard hotspotType="NEBRAIN" />
           <HotspotSelectionCard hotspotType="NEBRAOUT" />
         </Box>
-        <Box flexDirection="row" width="50%" height={191}>
+        <Box flexDirection="row" height={191}>
+          <HotspotSelectionCard hotspotType="Bobcat" />
```

Comment:
Can you move this to after `SYNCROBIT`. They should remain in the order they were added.

Reply:
Got it.

Reply:
Fixed

Reply:
I am not sure about the svg file, can you see it display properly on the selection screen?

Reply:
Thanks, yes the SVG is working as expected